tests/linker/ios/link all/InterfacesTest.cs

```diff
@@ -78,18 +82,18 @@ public void Bug10866 ()
 			F (new A ());
 
 			// Foo and Bar methods are both used on A and must be present
-			Assert.NotNull (typeof(A).GetMethod ("Foo", BindingFlags.Instance | BindingFlags.Public), "A::Foo");
-			Assert.NotNull (typeof(A).GetMethod ("Bar", BindingFlags.Instance | BindingFlags.Public), "A::Bar");
+			Assert.NotNull (type_a.GetMethod ("Foo", BindingFlags.Instance | BindingFlags.Public), "A::Foo");
+			Assert.NotNull (type_a.GetMethod ("Bar", BindingFlags.Instance | BindingFlags.Public), "A::Bar");
```

Comment:
Has something changed in the linker so the old and new code is not seen as identical ?
That smells like a breaking change, we can update the test not the existing (customer) code.

Reply:
The linker is now smarter so the test old code which used this approached to check for non-existence of the method actually forced the linker not to remove it.